src/client/entrypoints/main/board_tools.tsx

```diff
@@ -0,0 +1,125 @@
+import {BoardUpdateForm, NewBoardForm} from '_client/board_tools/board_form';
+import {BoardSelector, idSelector} from '_client/board_tools/board_selector';
+import * as UiUtil from '_client/common/ui_util';
+import {GameBoard} from '_client/game_board/controller/game_board';
+import {BoardModel} from '_client/game_board/model/board_model';
+import {BoardClient} from '_client/game_board/remote/board_client';
+import {DropdownSelector} from '_client/common/ui_components/dropdown';
+
+import React from 'react';
+import {loadBoard} from '_client/entrypoints/main/game_board';
+import ReactDOM from 'react-dom';
+import {Hideable} from '_client/common/ui_components/hideable';
+
+const NEW_BOARD_BUTTON = 'createNewBoard';
+const BOARD_FORM_STUB = 'createNewBoardFormStub';
+
+const MAIN_BOARD_STUB = 'mainBoard';
+
+const ACTIVE_SELECTOR_STUB = 'activeSelectorStub';
+const EDIT_SELECTOR_STUB = 'editSelectorStub';
+const EDITING_AREA_STUB = 'editingAreaStub';
+
+const SIDE_PANEL = 'sidePanelContent';
+
+const boardEditRow: React.CSSProperties = {
+  display: 'flex',
+  flexDirection: 'row',
+};
+
+export class EditingArea extends React.Component {
+  render(): JSX.Element {
+    return (
+      <div>
+        <div style={boardEditRow}>
+          <div id="activeSelectorStub"></div>
+          <div className="divider"></div>
+          <div id="editSelectorStub"></div>
+          <div className="divider"></div>
+        </div>
+        <br />
+        <div style={boardEditRow}>
+          <button id="createNewBoard" type="button" className="btn btn-primary">
+            Create New
+          </button>
+        </div>
+
+        <div id="createNewBoardFormStub" style={{zIndex: 30}}></div>
```

Comment:
This is hard-coded, so no need for the {{ }}

Reply:
Huh, I would think you could just do something like style="z-index: 30" like this (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/style). 

To be honest I don't think it matters one way or the other. Eventually it should live in a separate css file, and it's probably equivalent either way

Reply:
Got it - I think moving the styles out will have to be a separate PR, but definitely will clean that up eventually.